benchmark.rake

```diff
@@ -146,4 +146,41 @@ namespace :benchmark do
 
     puts 'TASK END: benchmark:put-metrics'
   end
+
+  desc 'Convert benchmark_report.json into RoadRunner compatible results.json'
+  task 'roadrunner', [:commit_id] do |_, args|
+    puts 'TASK START: benchmark:roadrunner'
+
+    require 'json'
+    require_relative 'roadrunner'
+
+    if File.exist?('benchmark_report.json')
```

Comment:
Guard clauses make this more readable. If the file doesn't exist, print and exit, then the rest of the logic that follows does not need to be inside an if statement.